gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/DistTarTask.groovy

```diff
@@ -51,7 +51,7 @@ class DistTarTask {
 
             from("${project.projectDir}/service") {
                 into "${archiveRootDir}/service"
-                exclude("**/bin/*")
+                exclude("bin/*")
```

Comment:
can you add a test illustrating what went wrong in https://github.com/palantir/sls-packaging/issues/247 ?

Reply:
The problem I observed was that this folder in my sls-packaged tarball was stripped empty: `spark-history-0.7.0-rc5-1-g52b7787/service/spark/spark-dist-2.3.0-palantir.13-hadoop-2.8.0-palantir.8/bin/`